superstake-ui/src/hooks/useEstimateApr.ts

```diff
@@ -112,23 +116,52 @@ const useEstimateApr = ({
 		superStakeLstDeposit * lstAprFromApy * lstMetrics.priceInSol;
 	const solInFromDeposit =
 		superStakeLstDeposit * lstDepositRate.toNum() * lstMetrics.priceInSol;
-	const leveragedStakingAPR =
+	const leveragedLstApr =
 		solInFromStaking / (initialLstDeposit * lstMetrics.priceInSol);
 	const solOut = solBorrowAmount * solBorrowRate.toNum();
 	const solRewardsPerYear = solInFromStaking + solInFromDeposit - solOut;
-
-	const projectedApr =
+	const lstNetProjectedApr =
 		solRewardsPerYear / (initialLstDeposit * lstMetrics.priceInSol);
+	const leveragedBorrowRate = solOut / (initialLstDeposit * lstMetrics.priceInSol);
+	const leveragedDepositRate = solInFromDeposit / (initialLstDeposit * lstMetrics.priceInSol);
+
+	// Add emissions APR on top because they are airdropped separately
+	// Not subject to the borrow rate
+	const emissionsAprFromApy = lstMetrics.emissionsApy ? 
+		aprFromApy(lstMetrics.emissionsApy, Math.floor(365 / 2)) / 100 : 0;
```

Comment:
It's a rough estimate of the number of epochs in a year, although the BLZE distributions are actually once a week. Not sure how solblaze is calculating the apy themselves that the api returns because BLZE isn't automatically compounding, hmm...